src/main/webapp/js/build-message-div.js

```diff
@@ -25,7 +25,9 @@ function buildMessageDiv(message){
   }
   const bodyDiv = document.createElement('div');
   bodyDiv.classList.add('message-body');
-  bodyDiv.innerHTML = SimpleMDE.prototype.markdown(message.text.replace('&gt;', '>')); // Allow quotes
+  var renderedHtml = SimpleMDE.prototype.markdown(message.text.replace('&gt;', '>')); // Allow quotes
+  renderedHtml = renderedHtml.replace(/<img /g, '<img class="w-100 p-1" ');
```

Comment:
what if the image already has a class attribute, potentially located somewhere else?

Reply:
I understand the issue, but it shouldn't happen because it seems that SimpleMDE renders the images without a class. I will take a closer look to see if its a problem next time, but will merge this first so that the change is implemented for the video recording.